src/PaymentForm.tsx

```diff
@@ -14,14 +16,30 @@ export default function PaymentForm({
     [],
   )
 
+  if (!configuration?.recaptchaPublicKey) {
+    console.error(
+      'OneBlinkForms.renderPaymentForm() cannot render a payment form without a reCAPTCHA configured on the "formsAppId" or "formsAppEnvironmentId" passed in.',
+      configuration,
+    )
+    return (
+      <div className="has-text-centered">
+        <h3 className="title is-3">Error Loading Configuration</h3>
+        <p className="content has-text-danger">
+          Insufficient configuration to complete payment. Please the site
+          administrators to rectify the issue.
```

Comment:
```suggestion
          Insufficient configuration to complete payment. Please the contact site
          administrators to rectify the issue.
```
or whatever word fits best